client/purplecaffeine/core.py

```diff
@@ -188,12 +198,11 @@ def export_to_shared_file(self, path) -> str:
         Returns:
             Full path of the file
         """
-        with open(
-            os.path.join(path, self.name + ".json"), "w", encoding="utf-8"
-        ) as trial_file:
+        filename = os.path.join(path, f"{self.uuid}.json")
```

Comment:
The goal here was to be able to share file with the name of the experiment so you can send a file with a good name to anyone